README.md

```diff
@@ -453,6 +456,55 @@ your adapter e.g.
 root of the cloned repository.
 
 
+## TypeScript and other transpiled languages support
```

Comment:
Info about this feature also should be added to CHANGELOG.md to section:

```md
## [Unreleased][]

### Added

```

Reply:
Which version will it be?

Reply:
It should be "Unreleased" section but there is not obvious changelog linter requirement related to this, so let me do that.